ffi/derive/src/lib.rs

```diff
@@ -173,7 +173,7 @@ pub fn ffi_export(_attr: TokenStream, item: TokenStream) -> TokenStream {
                 abort!(item.sig.generics, "Generics are not supported");
             }
 
-            let fn_descriptor = FnDescriptor::from(&item);
+            let fn_descriptor = FnDescriptor::from_fn(&item);
```

Comment:
```suggestion
            let fn_descriptor = FnDescriptor::fn_from(&item);
```

`from_fn` => From Function. 
`fn_from` => The function `from`.

Reply:
In this case we want `from_fn`, because we are creating `FnDescriptor` from freestanding function, likewise we have `from_impl_method` when we creating descriptor from method inside `impl` block.

Reply:
In that case `desc_from_fn`?

Reply:
I don't mind but don't full invocation `FnDescriptor::from_fn` is descriptive enough?